resources/js/mixins/url.js

```diff
@@ -0,0 +1,354 @@
+import Constants from '../constants';
+
+export default {
+    methods:{
+        /*
+         * Plentymarkets method for getting url params
+         * @returns {Object}
+         */
+        getUrlParams(urlParams) {
+            if (urlParams) {
+                var tokens;
+                var params = {};
+                var regex = /[?&]?([^=]+)=([^&]*)/g;
+
+                urlParams = urlParams.split("+").join(" ");
+
+                // eslint-disable-next-line
+                while (tokens = regex.exec(urlParams)) {
+                    params[decodeURIComponent(tokens[1])] = decodeURIComponent(tokens[2]);
+                }
+
+                return params;
+            }
+
+            return {};
+        },
+
+        /*
+         * FINDOLOGIC method for getting url params
```

Comment:
Same issue as with `getUrlParams`. It it still not clear what this function actually does.